resin/datadog_checks/resin/data/conf.yaml.example

```diff
@@ -0,0 +1,170 @@
+
+    ## @param name - string - optional
+    ## Set your instance name
+    #
+    # name: resin_instance
+
+    ## @param java_bin_path - string - optional
+    ## java_bin_path should be set if the agent cannot find your java executable
+    #
+    # java_bin_path: <JAVA_PATH>
+
+    ## @param java_options - string - optional
+    ## List of Java JVM options.
+    #
+    # java_options: "-Xmx200m -Xms50m"
+
+    ## @param trust_store_path - string - optional
+    ## trust_store_path should be set if "com.sun.management.jmxremote.ssl" is
+    ## set to true on the target JVM.
+    ## It's the path to your trusted store
+    #
+    # trust_store_path: <TRUSTSTORE.JKS_PATH>
+
+    ## @param trust_store_password - string - optional
+    ## `trust_store_password` should be set if "com.sun.management.jmxremote.ssl" is
+    ## set to true on the target JVM. It's the password for your TrustStore.jks file.
+    #
+    # trust_store_password: <PASSWORD>
+
+    ## @param key_store_path - string - optional
+    ## key_store_path should be set if "com.sun.management.jmxremote.ssl.need.client.auth"
+    ## is set to true on the target JVM.
+    ## It's the path to your key store.
+    #
+    # key_store_path: <KEYSTORE.JKS_PATH>
+
+    ## @param key_store_password - string - optional
+    ## key_store_password should be set if "com.sun.management.jmxremote.ssl.need.client.auth"
+    ## is set to true on the target JVM.
+    ## It's the password for your KeyStore.jks file.
+    #
+    # key_store_password: <PASSWORD>
+
+    ## @param rmi_registry_ssl - boolean - optional
+    ## Whether or not the agent should connect to the rmi registry using ssl.
+    ## should be set to true if "com.sun.management.jmxremote.registry.ssl" is set to true on the target JVM.
+    #
+    # rmi_registry_ssl: false
+
+    ## @param tags - list of key:value element - optional
+    ## List of tags to attach to every metric, event and service check emitted by this integration.
+    ##
+    ## Learn more about tagging: https://docs.datadoghq.com/tagging/
+    #
+    # tags:
+    #   - <KEY_1>:<VALUE_1>
+    #   - <KEY_2>:<VALUE_2>
+
+init_config:
+  ## @param is_jmx - boolean - required
+  ## Whether or not this file is a configuration for a JMX integration.
+  #
+  is_jmx: true
+
+  ## @param collect_default_metrics - boolean - required
+  ## Whether or not the check should collect all default metrics for this integration.
+  #
+  collect_default_metrics: true
+
+  ## @param conf - list of objects - required
+  ## List of metrics to be collected by the integration
+  ## Read http://docs.datadoghq.com/integrations/java/ to learn how to customize it
+  ## Agent 5: Customize all your metrics below
+  ## Agent 6: The default metrics to be collected are kept in metrics.yaml, but you can still add your own metrics here
+  #
+  conf:
+    - include:
+      domain: resin
+      type: ConnectionPool
+      attribute:
+        ConnectionCount:
```

Comment:
Hi, I noticed the metrics I'm getting in the test are not these but some with same name in ClusterServer instead of ConnectionPool. Would it make sense to include those metrics too? Otherwise they are collected with a 'jmx.*' prefix

Reply:
I believe its because no connections are being used or defined in this example.

Reply:
Yes, that makes sense. But would it make sense to set aliases also for `ClusterService`metrics? and then maybe rename duplicate names to be `resin.connection_pool.connection_active_count` and `resin.cluster_server.connection_active_count`

Reply:
I am not familiar with cluster service. I have not used it but I do know that on startup resin tries to talk to an internal mysql database.  We have seen this show up on APM but have not used it.  For now I would recommend leaving it off unless someone else feels its necessary.

Reply:
Fair enough, although those metrics are still going to get collected (with the jmx prefix). I still think is worth it to rename the aliases to `resin.connection_pool.` to avoid name clashes if someone wants to add the other metrics later.

Reply:
I see what you mean now that seems fair